### ISSUE TITLE
Taking fall damage after teleport to island, if you were falling.

### ISSUE THREAD

**Issue by lolsteamroller** [BentoBoxWorld/BSkyBlock#299]
#### Describe the bug

Teleporting to island while in motion applies that motion, resulting in fall damage taken.

#### Steps to reproduce the behavior

Go to edge of the island, jump off and immediately use any of the aliases for teleporting back to island. If immediate, it will take some fall damage, however, once you accelerate, the damage will result in death.

#### Expected behavior

Teleporting should just teleport you, without taking fall damage, no matter the situation you were in, or it should be found in config to modify. Only found preventing falling, but there were no clear flags on this case.

##### BentoBox Version (Mandatory)

BentoBox version: 1.12.0 / Same BSkyBlock - tested on Paper 1.15.2 too, Paper 1.14.4 + this below.
```
[05:16:30] [Server thread/INFO]: Running SPIGOT 1.14.4.
[05:16:30] [Server thread/INFO]: BentoBox version: 1.12.0
[05:16:30] [Server thread/INFO]: Database: JSON
[05:16:30] [Server thread/INFO]: Loaded Game Worlds:
[05:16:30] [Server thread/INFO]: bskyblock_world (BSkyBlock): Overworld, Nether, The End

```

#### Plugins (Optional)

Only pure BentoBox + these
```
[05:16:30] [Server thread/INFO]: Loaded Addons:
[05:16:30] [Server thread/INFO]: Biomes 1.7.0 (ENABLED)
[05:16:30] [Server thread/INFO]: BSkyBlock 1.12.0 (ENABLED)
[05:16:30] [Server thread/INFO]: Challenges 0.8.0 (ENABLED)
[05:16:30] [Server thread/INFO]: ControlPanel 1.7.0 (ENABLED)
[05:16:30] [Server thread/INFO]: IslandFly 1.7.3 (ENABLED)
[05:16:30] [Server thread/INFO]: Limits 1.9.4 (ENABLED)
[05:16:30] [Server thread/INFO]: MagicCobblestoneGenerator 1.7.0.2 (ENABLED)
[05:16:30] [Server thread/INFO]: TwerkingForTrees 1.3.0 (ENABLED)
[05:16:30] [Server thread/INFO]: VoidPortals 1.5.0.0 (ENABLED)
[05:16:30] [Server thread/INFO]: Warps 1.9.0 (ENABLED)
```

**Comment by tastybento** [comment]
This is a feature request.

**Comment by lolsteamroller** [comment]
> This is a feature request.

What do you mean? Having played ASkyBlock - I did not see such behaviour. 
When you type /is go - you teleported in one piece. Is dying after teleporting after jumping expected behaviour?

**Comment by tastybento** [comment]
Yes, absolutely. If you are falling and teleport and hit the ground then you will die. Players have often set their home in one block of water as a way to prevent that, but the standard behavior is to go splat. However, your request is a reasonable one for a config option. It take away a lot of the fear in the game if you can escape just by teleporting, but it can be an option.

**Comment by lolsteamroller** [comment]
Alright, thanks for explanation, I think I was playing on some servers, where this was fixed through other means then. Makes sense, actually, in the recent server, I saw my girlfriend put home in the pond, never made sense why.

I think other servers mods also made /sethome reject if you were in the air, doing the tests here, it seems, that if you are midair - it will reset to the sign / default position, and not give any errors.

Thanks for explanation...
